h2o-py/tests/pyunit_utils/utilsPY.py

```diff
@@ -3306,6 +3306,54 @@ def check_sorted_2_columns(frame1, sorted_column_indices, prob=0.5, ascending=[T
                                                                                              "row {2}: {3}".format(rowInd, frame1[rowInd,colInd],
                                                                                                                    rowInd+1, frame1[rowInd+1,colInd])
 
+def check_sorted_2_columns(frame1, sorted_column_indices, prob=0.5, ascending=[True, True]):
```

Comment:
this does not seem to be used, should it be removed?

Reply:
It is used here: h2o-py/tests/testdir_munging/pyunit_pubdev_4870_sort_bug_pubdev_4404_desc.py

Reply:
It is used in this test: h2o-py/tests/testdir_munging/pyunit_pubdev_4870_sort_bug_pubdev_4404_desc.py

Reply:
strange, then why does it show up here as new code? was it moved from somewhere?

Reply:
I might have moved it.  Not sure.